src/main/resources/config/application.yaml

```diff
@@ -0,0 +1,42 @@
+spring:
+  application:
+    name: dynamic-security-analysis-server
+
+  cloud:
+    function:
+      definition: consumeRun;consumeCancel
+    stream:
+      bindings:
+        consumeRun-in-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.run
+          group: dsaGroup
+          consumer:
+            concurrency: 2
+        publishRun-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.run
+        publishResult-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.result
+        consumeCancel-in-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.cancel
+        publishCancel-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.cancel
+        publishStopped-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.stopped
+        publishFailed-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.failed
+        publishCancelFailed-out-0:
+          destination: ${powsybl-ws.rabbitmq.destination.prefix:}dsa.cancelfailed
+      output-bindings: publishRun-out-0;publishResult-out-0;publishCancel-out-0;publishStopped-out-0;publishFailed-out-0;publishCancelFailed-out-0
+
+powsybl:
```

Comment:
We rather set this in the Service

Reply:
https://github.com/gridsuite/loadflow-server/blob/b3998ffdafee0ef547ab665a00459b634d951163/src/main/java/org/gridsuite/loadflow/server/service/LoadFlowWorkerService.java#L125

Reply:
OK

Reply:
DONE in https://github.com/gridsuite/dynamic-security-analysis-server/pull/1/commits/9433ce504076cf82eeab5658042569ef1619363c